vhdbuilder/packer/install-dependencies.sh

```diff
@@ -413,17 +440,29 @@ if [[ $OS == $UBUNTU_OS_NAME ]]; then
   sed -i 's/After=network-online.target/After=multi-user.target/g' /lib/systemd/system/motd-news.service
 fi
 
+# use the private_packages_url to download and cache packages
+if [[ -n ${PRIVATE_PACKAGES_URL} ]]; then
+  IFS=',' read -ra PRIVATE_URLS <<< "${PRIVATE_PACKAGES_URL}"
```

Comment:
we should probably reset IFS after overriding it here,

IFS_backup=$IFS; IFS=','
....
IFS=$IFS_backup

Reply:
since IFS and read are on the same line, IFS value before and after this line remains the same, don't have to backup and restore - something like - Prefixing a command with parameter assignment affects the environment of the command being executed.?
Or am i missing anything?

Reply:
ah, I guess you're right, just tested this locally